web/src/composables/useTransformTupleData.ts

```diff
@@ -0,0 +1,91 @@
+import { Ref, computed } from "vue"
+import { ModelType } from "@/domain/types"
+
+type FieldData = { [fieldName: string]: ModelType }
+
+function getTupleKey(parentKey: string, childKey: string) {
+  const sentenceCaseChildKey =
+    childKey.charAt(0).toUpperCase() + childKey.slice(1)
+  return `${parentKey}And${sentenceCaseChildKey}`
+}
+
+function getKeysFromTupleKey(combinedTupleKey: string) {
+  const parentKey = combinedTupleKey.substring(
+    0,
+    combinedTupleKey.indexOf("And")
```

Comment:
So `And` is used as a string-keyword her to mark this internally. In line 9 it is constructed and here we ask for it 🤔 I wonder if there might be a non-string way to do this? Not sure though. But if it has to be string, a more unique keyword could make sense? I could imagine accidentally an `And` appearing in one of the two keys and then the `indexOf()` would not work properly anymore.

Reply:
I see your point. Since we need IDs for the components, I cannot think of an alternative rn. 

A random `And` should hopefully not brake things, since we only call the function on tuples (see line 47)?

Reply:
good point! Since, we need the ids I cannot really thing of a good alternative rn. Maybe just check `And` only occurs once?

Reply:
resolved by [b6d115f](https://github.com/digitalservicebund/ris-backend-service/pull/295/commits/b6d115f69f8d903e8c493540395fa35b3f7582a2)